src/main/java/me/asleepp/SkriptItemsAdder/elements/types/Types.java

```diff
@@ -17,34 +21,61 @@ public class Types {
     public static Map<String, TexturedInventoryWrapper> inventoryMap = new HashMap<>();
 
     static {
-        Classes.registerClass(new ClassInfo<>(CustomBlock.class, "customblock")
-                .user("customblock")
-                .name("Custom Block")
-                .description("Represents a Custom/ItemsAdder Block")
-                .defaultExpression(new EventValueExpression<>(CustomBlock.class))
-                .since("1.5")
-                .parser(new Parser<CustomBlock>() {
-
+        Classes.registerClass(new ClassInfo<>(CustomItemType.class, "customitemtype")
+                .user("customitemtypes?")
+                .name("Custom Item Type")
+                .description("Represents a custom item type using aliases.")
+                .serializer(new Serializer<CustomItemType>() {
                     @Override
-                    public String toString(CustomBlock customBlock, int flags) {
-                        return customBlock.getId();
+                    public @NotNull Fields serialize(CustomItemType customItemType) {
+                        Fields fields = new Fields();
+                        fields.putObject("namespacedID", customItemType.getNamespacedID());
+                        return fields;
                     }
 
                     @Override
-                    public String toVariableNameString(CustomBlock customBlock) {
-                        return "customblock:" + customBlock.getId();
+                    public void deserialize(CustomItemType customItemType, @NotNull Fields fields) {
+                        try {
+                            customItemType.setNamespacedID((String) fields.getObject("namespacedID"));
+                        } catch (java.io.StreamCorruptedException e) {
+                            e.printStackTrace();
+                        }
                     }
 
                     @Override
-                    public CustomBlock parse(String s, ParseContext context) {
-                        return CustomBlock.getInstance(s);
+                    public boolean mustSyncDeserialization() {
+                        return true;
                     }
 
                     @Override
-                    public boolean canParse(ParseContext context) {
+                    public boolean canBeInstantiated() {
                         return true;
```

Comment:
I'm pretty sure you have to have a nullary constructor for this to work. That is:
```java
public CustomItemType() { }
```
If you can't do that, set this to false, and there should be another deserialize method where you return the CustomItemType object